# machine name short
string name_short

# machine name long
string name_long

# team 0=CYAN 1=MAGENTA 2=NO_TEAM
int8 team

# machine position and orientation
float32 x
float32 y
float32 yaw
